conformance/lang/values-types-and-variables/array.balt

```diff
@@ -0,0 +1,99 @@
+Test-case: output
+Description: Test valid array types and values
+Label: array
+
+const LENGTH = 2;
+
+public function main() returns () {
+    int[5] a = [1, 2, 3, 4, 5];
+    io:println(a); // @output [1,2,3,4,5]
```

Comment:
Why are there no spaces (eg: `[1, 2, 3, 4, 5]`), is this format defined in the spec ?

Reply:
There are no spaces shown in the table here: https://ballerina.io/spec/lang/2021R1/#ToString